layouts/partials/footer.html

```diff
@@ -27,7 +27,7 @@ <h4>Permalinks</h4>
         <div class="footer__3">
             <h4>Follow Us</h4>
             <ul class="footer__socials">
-                <li><a href="{{ .Site.Author.discourse }}"><i class="uil uil-comment-alt-lines"></i></a></li>
+                <li><a href="{{ .Site.Author.discourse }}"><i class="uil uil-comment-alt-lines"></i> alt="Link to discourse"</a></li>
```

Comment:
not sure this will do what we want it to do. 

Potentially useful references:
* https://axesslab.com/icons-ruining-interfaces/ (here we use logos instead of icons so while it is possible someone doesn't know what they mean, they are not ambiguous I think).
* https://stackoverflow.com/a/38431457 on how to add text with the icon but hide it so it is only read by screen readers